storagenode/peer.go

```diff
@@ -248,6 +250,14 @@ func New(log *zap.Logger, full *identity.FullIdentity, db DB, revocationDB exten
 			peer.DB.PieceSpaceUsedDB(),
 		)
 
+		peer.Storage2.TrashChore = pieces.NewTrashChore(
+			log.Named("pieces:trashchore"),
+			24*time.Hour,   // choreInterval: how often to run the chore
+			7*24*time.Hour, // trashExpiryInterval: when items in the trash should be deleted
```

Comment:
Should these options be configurable? PR looks good to me! I don't think a merge should be pending on the answer of this question :)